packages/contracts-bedrock/src/dispute/BlockHashOracle.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import "../libraries/DisputeTypes.sol";
+import "../libraries/DisputeErrors.sol";
+
+/// @title BlockHashOracle
+/// @notice Stores a map of block numbers => block hashes for use in dispute resolution
+contract BlockHashOracle {
+    /// @notice Maps block numbers to block hashes
+    mapping(uint256 => Hash) internal blockHashes;
```

Comment:
internal prefixed with `_`?

Reply:
Yeah is that in our style guide? Not opposed, but don't think this style is defined.

Reply:
Not sure

Reply:
maybe its for internal functions, i think its underspecified in this case. not super opinionated